src/it/resources/org/checkstyle/suppressionxpathfilter/methodlength/SuppressionXpathRegressionMethodLengthOne.java

```diff
@@ -0,0 +1,27 @@
+package org.checkstyle.suppressionxpathfilter.methodlength;
+
+public class SuppressionXpathRegressionMethodLengthOne {
+    protected SuppressionXpathRegressionMethodLengthOne() { // warn
+        // a line
+        // a line
+        // a line
+        // a line
+        // a line
+        // a line
+        // a line
+        // a line
+
+    }
+
+    private void methodOne(){
```

Comment:
Firstly I tried having violations in this method also. Which makes two violations in this file.
Then when generating xpath suppression for both lines seperately and when running, it was giving error that it expected only one.
Is it that expected xpath query should be of only one violation ?

Reply:
Please share examples, it is hard to understand what you are describing above.

Reply:
The method does not give any violation because the length is under the maximum limit. If you are trying to say something else, please share details.

Reply:
So I wanted to clarify if a file has multiple violations, do we have to add xpath queries for all the violations in `expectedXpathQueries`  ?

Because when I tried the same in this case by making `methodOne` exceeding the number of lines and adding the queries, it gave me an error for diff in actual and expected. Which seems a bit strange to me.

Reply:
@TanayMorakhia Sorry for the delay. I checked, and yes you are correct, we are not supporting verifying multiple violation suppressions in XPath testing module. I opened https://github.com/checkstyle/checkstyle/issues/14435 for it. The current tests look fine to me.